src/app/profile/profile-routing.module.ts

```diff
@@ -15,8 +15,10 @@ const routes: Routes = [
     component: ProfileComponent,
     children: [
       { path: '',          component: OverviewComponent },
-      { path: 'spaces',    loadChildren: './collaboration/collaboration.module#CollaborationModule' },
+      { path: 'spaces',    loadChildren: './spaces/spaces.module#SpacesModule' },
       { path: 'resources', loadChildren: './resources/resources.module#ResourcesModule' },
+      { path: '', loadChildren: './resources/resources.module#ResourcesModule' }
```

Comment:
What are you trying to do here without the path? The first line with no path is the default going to the Overview page.